rs/p2p/artifact_downloader/src/fetch_artifact/download.rs

```diff
@@ -159,14 +160,14 @@ impl<Artifact: PbArtifact> FetchArtifact<Artifact> {
             build_axum_router(pool_clone),
         )
     }
-    /// Waits until advert resolves to fetch. If all peers are removed or bouncer value becomes Unwanted `Aborted` is returned.
+    /// Waits until advert resolves to fetch. If all peers are removed or bouncer value becomes Unwanted, false is returned.
```

Comment:
Do we really return false here if all peers are removed?
```suggestion
    /// Waits until advert resolves to wanted. If the bouncer value becomes Unwanted, false is returned.
```